docs/geckoboard.md

```diff
@@ -0,0 +1,99 @@
+# Geckoboard
+
+The service uses [Geckoboard](https://www.geckoboard.com/) to present summary
+information on the current state and overall performance of the service. As part
+of this, the application sends updates on the state of claims to a
+[dataset](https://support.geckoboard.com/hc/en-us/articles/223190488-Guide-to-using-datasets).
+The data are then sliced and aggregated in Geckoboard for reporting purposes.
+
+## API key
+
+The API key for Geckoboard can be found in the
+[Geckoboard account details](https://app.geckoboard.com/account/details). It
+should be placed in the `GECKOBOARD_API_KEY` environment variable.
+
+## The data
+
+Geckoboard limits datasets to ten columns. We send the following ten for each
+claim, updated for all claims when a claim moves between states, and for
+[all unchecked claims each morning](../app/jobs/geckoboard/update_unchecked_claims_job.rb).
+These are defined in code in
+[`models/claim/geckoboard_dataset.rb`](../app/models/claim/geckoboard_dataset.rb)
+
+### 1. Claim reference
+
+The human-readable reference for the claim. This is the unique key for the
+dataset, any updates sent to the dataset with the same reference as an existing
+datapoint will be merged.
+
+### 2. Policy name
+
+A string of the policy which the claim was submitted under.
+
+### 3. Submitted at
+
+The date and time the claim was submitted.
+
+### 4. SLA Status
+
+A string representing the state of the claim relative to its SLA deadline. This
+can be one of three values:
+
+- `ok` - The claim is not approaching its check deadline
+- `warning` - The claim is within the warning period of its check deadline
+- `passed` - The claim has passed its check deadline
+
+### 5. Check status
+
+A string representing the current status of a claim with regard to checking. It
+can be one of three values:
+
+- `unchecked` - The claim has not yet been checked
+- `approved` - The claim has been approved
+- `rejected` - The claim has been rejected
+
+From these it is possible to infer if a claim has been checked or not, as well
+as the outcome of that check.
+
+### 6. Checked at
+
+If a claim has been checked, the date and time a check decision about the claim
+was made.
+
+If a claim has not been checked this will be the result of the
+`placeholder_date_for_nil_value` function in the model.
```

Comment:
Might be worth explaining _why_ we do that, as it might seem a bit weird without any context

Reply:
This could be done via a comment in the code actually